packages/resolve-scripts/src/server/express.js

```diff
@@ -59,10 +60,14 @@ app.post(`${rootDirectory}/api/commands`, (req, res) => {
     });
 });
 
-app.use(
-    `${rootDirectory}${STATIC_PATH}`,
-    express.static(path.join(__dirname, '../../dist/static'))
-);
+const staticMiddleware = process.env.NODE_ENV === 'production'
+    ? express.static(path.join(__dirname, '../../dist/static'))
+    : (req, res) => {
+        var newurl = 'http://localhost:3001' + req.path;
```

Comment:
Have you checked that it works? Can webpack-dev-server give not only bundle file but any static file from user static folder?